packages/block-library/src/gallery/gallery-image-style.native.scss

```diff
@@ -0,0 +1,169 @@
+@function get-gallery-image-container-height() {
```

Comment:
why not use variables instead of functions?

$gallery-image-container-height: 150px;

Reply:
I'll change this.. originally I tried to find a way to avoid polluting the global scope, but I guess with our transformer that is not necessary :smiley:

Reply:
:heavy_check_mark: Updated.